pack1/Laba1_5/Makefile

```
@@ -0,0 +1,102 @@
+CС=gcc
+CFLAGS= -Wall -Werror -Wextra
+
+SOURCES=task_1_5.c # Тут вы перечислятете файлы с вашими лабами по 1 файлу на задание
+TEST_TEMPLATES_DIR=/home/admin1/FundAlg3/Laba_1/Laba1_5# Это директория с шаблонами тестов (если вы их разместите в той же директории что и лабы пишите ./)
+EXECUTABLES=$(SOURCES:.c=.a)
```

Заментить на относительные пути
```suggestion
TEST_TEMPLATES_DIR=../../test_templates# Это директория с шаблонами тестов (если вы их разместите в той же директории что и лабы пишите ./)
```